### ISSUE TITLE
streetpass

### ISSUE THREAD

**Issue by eeelkababs** [togenyan/Yo-kai_Editor_Getto#11]
is there any way to edit streetpass or the time play with other game version?

streetpass is almost dead where i'm from, and even some people who have access to area with it, they are complaining that  unlocking the popularity and xl oni orb size is impossible at the moment :(

if Nintendo didn't kill homepass the unlocking aura issue won't be a big problem , even now finding pandanoko is nearly impossible.

**Comment by gurglebutt** [comment]
No, you can't edit street pass and the time because street pass requires you to pass by a real person and yokai watch blasters uses the 3ds clock in real time.
